.changes/next-release/bugfix-caf370b9-c943-4ece-a4c0-81bcefa7a4f5.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "bugfix",
+  "description" : "Q dev add error messages for upload URL expiry"
```

Comment:
probably should make this description a little easier to understand or more vague

> Amazon Q feature dev: Add error message for connection timeout

Reply:
It is for upload URL explicitly